docker/build_geant4.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-set -e
+set -ex
 
 source ${docker_env}
 
 rm -rf ${geant4_build_dir}/bld ${geant4_install_dir}
 mkdir -p ${geant4_build_dir}/bld
 cd ${geant4_build_dir}
-wget http://geant4.cern.ch/support/source/geant4.${geant4_version}.tar.gz
+wget https://geant4.cern.ch/support/source/geant4.${geant4_version}.tar.gz --no-check-certificate
```

Comment:
In #660 they mention verifying our file against a checksum. Where do we stand on adding that?

Reply:
did not found any checksum for geant4 tarball :(

Reply:
We could generate a checksum against the version we are pulling

Reply:
make sense !